doc/sphinx/contributing.rst

```diff
@@ -84,22 +84,26 @@ Required Development Tools
    distributed versioning control system git_ and a GitHub account to fork the
    `espressomd/espresso <https://github.com/espressomd/espresso>`_ repository
 
--  To build the sphinx documentation, you will need the Python packages listed
-   in :file:`requirements.txt` in the top-level source directory. To install
-   them, use:
+-  To build the user documentation, you will need Sphinx_.
 
-   .. code-block:: bash
+-  To build the tutorials, you will need LaTeX and Jupyter.
 
-      pip install --upgrade --user -r requirements.txt
+-  To build the core documentation, you will need Doxygen_.
 
-   Note that some distributions now use ``pip`` for Python3 and ``pip2`` for
-   Python2.
+All of these tools should be easy to install on most Unix operating systems.
 
--  To build the tutorials, you will need LaTeX and Jupyter.
+You can find all Python dependencies of |es| in :file:`requirements.txt` in the
+top-level source directory. Several optional packages for graphics, external
+devices and continuous integration (CI) are not strictly essential and can be
+safely removed if you're planning on installing dependencies via ``pip``:
 
--  To compile the Doxygen code documentation, you will need the tool doxygen_.
+.. code-block:: bash
+
+   pip install --upgrade --user -r requirements.txt
```

Comment:
Change this to `pip3`. That seems to be the default name nowadays.

Reply:
Thanks!

Reply:
`python3 -m pip` is also a possibility.